### PR TITLE
docs: update timestamp spec

### LINKED ISSUE #303: Revocation check control for timestamp verification
> **Revocation check** : Guarantees that the TSA identity is still trusted at verification time. Events such as key or system compromise can cause a previously trusted TSA identity to become untrusted. It is always enforced when timestamp countersignature verification is triggered.

Shouldn't this be governed by [revocation](https://github.com/notaryproject/specifications/blob/main/specs/trust-store-trust-policy.md#signature-verification-details) configuration in trust policy ?

_Originally posted by @priteshbandi in https://github.com/notaryproject/specifications/pull/290#discussion_r1646655747_

### PR DESCRIPTION
**Pull request by ghost**
Resolves #303

### CODE REVIEW

**Review by shizhMSFT — APPROVED** [review]
LGTM

File: specs/trust-store-trust-policy.md [hunk 1]
```diff
@@ -201,7 +203,8 @@ Users who consume signed artifacts from OCI registries, or signed arbitrary blob
             "registryScopes": [ "*" ],
             "signatureVerification": {
               "level" : "strict",
-              "verifyTimestamp": "afterCertExpiry"  // Only verify any timestamp countersignature when codesigning certificate chain has expired
+              "verifyTimestamp": "afterCertExpiry",  // Only verify any timestamp countersignature when codesigning certificate chain has expired
+              "skipTimestampRevocationCheck": true   // Skip the TSA certificate chain revocation check during timestamp verification
```
Comment (FeynmanZhou) [inline specs/trust-store-trust-policy.md:207]: If `skipTimestampRevocationCheck` is a bool option, does it support audit mode and log each timestamp signature verification check?

Comment (ghost) [reply specs/trust-store-trust-policy.md:207]: In what situation do we want the timestamp revocation check but continue on failures? If user does not care about the result why not just skip it and save the roundtrip?

(If we do have such use cases, we can update the field to take a string just like `verifyTimestamp`)

**Comment by Two-Hearts** [comment]
Based on 7/8/24 community meeting, will close this PR due to issue #303 has been moved to Future milestone.

**Comment by ghost** [comment]
Based on 7/8/24 community meeting, closing this PR due to issue https://github.com/notaryproject/specifications/issues/303 has been moved to Future milestone.

**Pull request closed without merging**